bin/tigmint-molecule

```diff
@@ -62,7 +61,7 @@ class MolecIdentifier:
     """Group molecules into barcodes"""
 
     def __init__(self):
-        """Constructor, identifies molecules based on inter-arrival time threshold."""
+        """Constructor"""
```

Comment:
```suggestion
        """Constructor"""
```
I think we could keep this comment in? We're still using inter-arrival distances to infer molecules, even though we aren't saving the inter-arrivals anymore?